chapters/developmentProcess.typ

```diff
@@ -1,34 +1,54 @@
 = Development Process <development_process>
 
-== Workflow
+Within this section, it is explained how the working environment was setup and what tools were used for development.
+@workflow documents what the work process looked like. 
+In @setup, it is described how the project was setup for Windows and Linux.
+
+== Workflow <workflow>
+
+As the LLVM project, to which the new refactoring features should be contributed, is on GitHub, it was decided to also work using GitHub.
+
+As on GitHub was already an existing pipeline from the original LLVM project itself, the decision came rather quickly, as otherwise there could have popped up a lot of different problems and issues.
```

Comment:
De Absatz obe dra fangt au scho mit "as" a.

Reply:
```suggestion
On GitHub a pipline already exists for the original LLVM project itself, the decision came rather quickly, as otherwise unexpected problems and issues could have occured.
```

Reply:
Langet das ned?

```suggestion
The LLVM project already had workflows set up on GitHub, so the decision was made to re-use them.
```